pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitRequestShippingTest.java

```diff
@@ -66,6 +72,13 @@ public static void doFirst() {
         notLegacyParticipant = TestParticipantUtil.createParticipantWithEsProfile(notLegacyParticipantGuid, profile, ddpInstanceDto);
         notLegacyParticipantGuid = notLegacyParticipant.getRequiredDdpParticipantId();
         participants.add(notLegacyParticipant);
+
+        Profile mimicNotHuridParticipant = new Profile();
```

Comment:
I suggest moving this to the test that uses it since it is not shared among tests.
That will make the test and this setup method easier to understand.